backend/auth-service/src/test/kotlin/nz/unitracker/backend/authservice/AuthServiceApplicationTests.kt

```
@@ -0,0 +1,11 @@
+package nz.unitracker.backend.authservice
+
+import nz.unitracker.backend.authservice.helper.AuthIntegrationTest
+import org.junit.jupiter.api.Test
+
+@AuthIntegrationTest
+class AuthServiceApplicationTests {
+    @Test
+    fun contextLoads() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)